qmlbind/include/qmlbind/qmlbind_global.h

```diff
@@ -51,9 +51,8 @@ typedef QJSValueIterator qmlbind_iterator;
 typedef QByteArray qmlbind_string;
 
 // Use shared pointer for widely referenced classes
-typedef std::shared_ptr<QmlBind::Interface> qmlbind_interface;
 typedef std::shared_ptr<QmlBind::MetaObject> qmlbind_metaobject;
-typedef std::shared_ptr<QmlBind::Exporter> qmlbind_exporter;
+typedef std::unique_ptr<QmlBind::Exporter> qmlbind_exporter;
```

Comment:
You're right, the unique_ptr here does not provide any features but only an unneccessary layer of indirection. I'm changing it. :+1: